Makefile

```diff
@@ -6,6 +6,15 @@ clean:
 	rm -rf build dist *egg-info ./__pycache__
 	find -name *.pyc -delete
 
+#
+# Triage search tool
+#
+prepare_search:
+	$(container_command) build . -t ticket-search-container -f Dockerfile.assisted-installer-deployment
```

Comment:
seems to just build the assisted-installer-deployment, nothing specific to the ticket-search tool

Reply:
It pretty much does, yes, I just wanted to keep this simple for the user. 
The target is "prepare_search" how we make that happen can be hidden from the user?

Reply:
Another thing worth noting is that I don't want to build this target using skipper, the whole point of this PR is that we don't use skipper. The makefile does not have a target that just builds this, so I think this is OK

Reply:
Why "make prepare_search" is better than "make image"? Not sure I follow

Reply:
Fair point on the naming convention, I have updated now.